internal/resources/ibm/is_instance.go

```diff
@@ -97,6 +98,62 @@ type CatalogInstance struct {
 	Metadata Metadata `json:"metadata"`
 }
 
+type ProfileMultipliers struct {
+	Cpu    decimal.Decimal
+	Memory decimal.Decimal
+}
+
+var multipliers = map[string]ProfileMultipliers{
+	"bx2": {
+		Cpu:    decimal.NewFromFloat(0.902054794596651),
+		Memory: decimal.NewFromFloat(1.14870234843351),
+	},
+	"bx2d": {
+		Cpu:    decimal.NewFromFloat(0.902054794596651),
+		Memory: decimal.NewFromFloat(1.14870234843351),
+	},
+	"cx2": {
+		Cpu:    decimal.NewFromFloat(0.902054794596651),
+		Memory: decimal.NewFromFloat(1.70926497079),
+	},
+	"cx2d": {
+		Cpu:    decimal.NewFromFloat(0.902054794596651),
+		Memory: decimal.NewFromFloat(1.70926497079),
+	},
+	"gx2": {
+		Cpu:    decimal.NewFromFloat(0.963755342547062),
+		Memory: decimal.NewFromFloat(0.902054794596651),
+	},
+	"mx2": {
+		Cpu:    decimal.NewFromFloat(0.963755342547062),
+		Memory: decimal.NewFromFloat(0.902054794596651),
+	},
+	"mx2d": {
+		Cpu:    decimal.NewFromFloat(0.963755342547062),
+		Memory: decimal.NewFromFloat(0.902054794596651),
+	},
+	"ux2d": {
+		Cpu:    decimal.NewFromFloat(2.40343479472332),
+		Memory: decimal.NewFromFloat(0.902054794596651),
+	},
+	"vx2d": {
+		Cpu:    decimal.NewFromFloat(1.39565917819994),
+		Memory: decimal.NewFromFloat(0.902054794596651),
+	},
+}
+
+func getProfileMultiplier(profile string) ProfileMultipliers {
+	splitProfile := strings.Split(profile, "-")
+	multiplier := ProfileMultipliers{Cpu: decimal.NewFromInt(1), Memory: decimal.NewFromInt(1)}
+	if splitProfile[0] != "" {
+		val, ok := multipliers[splitProfile[0]]
```

Comment:
if an unknown profile is encountered, the function should return a default multiplier of (1,1)

Reply:
```
	multiplier := ProfileMultipliers{Cpu: decimal.NewFromInt(1), Memory: decimal.NewFromInt(1)}
```
I think my code already does that

Reply:
you're right, I missed line 147. sorry.